wavelength-plugins/misc.js

```diff
@@ -487,15 +487,15 @@ exports.commands = {
 		if (!this.runBroadcast()) return false;
 		if (!target || target === 'help') return this.parse('/help wlssb');
 		let targetData = getMonData(toId(target));
-		if (['km', 'callie', 'krakenmare'].includes(toId(target))) return this.sendReplyBox(getMonData('callieagent1'));
+		if (['km', 'callie', 'krakenmeme'].includes(toId(target))) return this.sendReplyBox(getMonData('krakenmare'));
 		if (toId(target) === toId('c7')) return this.sendReplyBox(getMonData('c733937123'));
 		if (['des', 'deso'].includes(toId(target))) return this.sendReplyBox(getMonData('desokoro'));
 		if (['mos', 'electricz'].includes(toId(target))) return this.sendReplyBox(getMonData('mosmero'));
 		if (toId(target) === toId('prince')) return this.sendReplyBox(getMonData('wavelengthprince'));
 		if (toId(target) === toId('hiro')) return this.sendReplyBox(getMonData('hiroz'));
 		if (['hh', 'hoeen'].includes(toId(target))) return this.sendReplyBox(getMonData('hoeenhero'));
-		if (toId(target) === toId('[]')) return this.sendReplyBox(getMonData('arrays'));
-		if (['mech', 'mechsteelix'].includes(toId(target))) return this.sendReplyBox(getMonData('isteelx'));
+		if (toId(target) === toId('arrays')) return this.sendReplyBox(getMonData('volco'));
```

Comment:
why are we running toId on something that is an id?

Reply:
No idea tbh. I just edited stuff @HoeenCoder

Reply:
edit the toId out then